mail/protocols/imap/src/main/java/com/fsck/k9/mail/store/imap/RealImapConnection.kt

```diff
@@ -561,6 +562,12 @@ internal class RealImapConnection(
         }
     }
 
+    private fun sendClientIDIfSupported() {
+        if (hasCapability(Capabilities.ID) && settings.clientId != null) {
+            executeSimpleCommand("ID (" + settings.clientId + ")")
```

Comment:
When only handing an app name to the IMAP code, this needs to be changed to:
```suggestion
            val encodedAppName = ImapUtility.encodeString(settings.clientIdAppName)
            executeSimpleCommand("""ID ("name" $encodedAppName)""")
```